frameworks/Rust/actix/src/db_pg_direct.rs

```diff
@@ -132,7 +133,8 @@ impl PgConnection {
                 );
 
                 for w in &worlds {
-                    update.push_str(&format!("({}, {}),", w.id, w.randomnumber));
+                    let _ =
+                        write!(&mut update, "({}, {}),", w.id, rng.gen_range(1, 10_001));
```

Comment:
> Each World object must have its randomNumber field updated to a new random integer between 1 and 10000.

You need to update the `World` object, then you can get the value from it. It's also necessary as the new value needs to be returned in json.

Reply:
right. fixed.